llama_index/__init__.py

```diff
@@ -64,7 +64,34 @@
     PromptTemplate,
     SelectorPromptTemplate,
 )
-from llama_index.readers import SimpleDirectoryReader, download_loader
+from llama_index.readers import (
+    BeautifulSoupWebReader,
+    ChromaReader,
+    DeepLakeReader,
+    DiscordReader,
+    FaissReader,
+    GithubRepositoryReader,
+    GoogleDocsReader,
+    JSONReader,
+    MboxReader,
+    MilvusReader,
+    NotionPageReader,
+    ObsidianReader,
+    PineconeReader,
+    PsychicReader,
+    QdrantReader,
+    RssReader,
+    SimpleDirectoryReader,
+    SimpleMongoReader,
+    SimpleWebPageReader,
+    SlackReader,
+    StringIterableReader,
+    TrafilaturaWebReader,
+    TwitterTweetReader,
+    WeaviateReader,
+    WikipediaReader,
+    download_loader,
+)
```

Comment:
Oh, we actually don't want to import these here.

Was there a notebook that was importing from here? we should update it

Reply:
yes, `from llama_index import SummaryIndex, RssReader`

Reply:
we should update that example instead 👍🏻

Reply:
@logan-markewich there are a bunch of readers defined in this file, should we remove them from ```
__all__ = [***]```

Reply:
Yea, only simple directory reader and download_loader should be used and exported here